images/pull-kubernetes-bazel/runner

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+git clone https://github.com/kubernetes/test-infra
+./test-infra/jenkins/bootstrap.py \
+    --upload='gs://kubernetes-jenkins/pr-logs' \
+    --repo=k8s.io/kubernetes \
+    --pull=${PULL_REFS} \
```

Comment:
This feels a little odd...

I would prefer we a) make this a python program b) use argparse to specify --pull, etc and c) pass in JOB_NAME and PULL_REFS as arguments to the image.

AKA I would like to move away from magic environment variables to a more explicit interface using arguments.